app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -616,13 +616,20 @@ public function buildCart($quote, $items, $multipage)
         ///////////////////////////////////////////////////////////
         // Generate base cart data, quote, order and items related.
         ///////////////////////////////////////////////////////////
+        /** @var Mage_Catalog_Model_Product_Media_Config $productMediaConfig */
         $productMediaConfig = Mage::getModel('catalog/product_media_config');
+        /** @var Mage_Catalog_Helper_Image $imageHelper */
+        $imageHelper = Mage::helper('catalog/image');
+        $imagePlaceholder = $imageHelper->getPlaceholder();
         $cartSubmissionData = array(
             'order_reference' => $quote->getId(),
             'display_id'      => $quote->getReservedOrderId(),
             'items'           => array_map(
-                function ($item) use ($quote, $productMediaConfig, &$calculatedTotal) {
-                    $imageUrl = $productMediaConfig->getMediaUrl($item->getProduct()->getThumbnail());
+                function ($item) use ($quote, $productMediaConfig, &$calculatedTotal, $imagePlaceholder) {
```

Comment:
can you add helper method `getImageUrl($item)` and move all logic there?

Also please add unit tests